### ISSUE TITLE
APS 非重点复习课程之智能计算系统 | min hjemmeside

### ISSUE THREAD

**Issue by tinsir888** [tinsir888/tinsir888.github.io-gittalk#130]
https://tinsir888.github.io/posts/fac97af3.html 

 Intelligent Computing System is a combination of Deep Learning, Parallel Programming, Computer Organization and Computer Architecture.  Neural Network Basis Loss function L(w)=12∑i(H(xi)−yi)2=12∑i(wTx
